### PR TITLE
Reactify

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Reactify the map.

Fixes # 166027992

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# package.json is formatted by package managers, so we ignore it here
+package.json
+package-lock.json
```
Comment (kilemensi) [inline .prettierignore]: 1. Let `prettier` format everything.
2. Lets stick to `yarn` and `yarn-lock`

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none",
```
Comment (kilemensi) [inline .prettierrc]: I love trailing commas, especially in arrays. e.g.
```javascript
...
state = [
  'ready',
  'set',
  'go',
]
....
```

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@import url('https://fonts.googleapis.com/css?family=Oswald:300,400,500');
```
Comment (kilemensi) [inline src/index.css]: What happened to using themes?

**Comment by nerdlet** [comment]
Hi @karimkawambwa Maybe a QQ: Is there any particular reason why we are using typescript with react?

**Comment by karimkawambwa** [comment]
> Hi @karimkawambwa Maybe a QQ: Is there any particular reason why we are using typescript with react?

@nerdlet to enforce standards and catch bugs at compile time. Also typescript makes it much simpler to have interface than the proptype definitions.

**Comment by nerdlet** [comment]
@karimkawambwa Looks good to me! 
Nice work!  
Wouldnt have used typescript otherwise...but noticed that it works so well with  ` react-mapbox-gl  ` 
cc @kilemensi

**Comment by nerdlet** [comment]
@karimkawambwa Also, not sure if the map is showing on your end, am getting this on my end

image

**Comment by karimkawambwa** [comment]
@nerdlet use the developer key for your map, the one in production is restricted

File: src/components/Mapbox.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        found.push(filter[1]);
+      } else {
+        filtersToApply.push(['in', ...filter]);
+      }
+    });
+    setBlastLinesFilter(filtersToApply);
+  }, [filters]);
+
+  return (
+    <Map
```
Comment (nerdlet) [inline src/components/Mapbox.tsx]: @karimkawambwa Just a heads up ....getting this `Style prop value must be an object  react/style-prop-object` on the console. Fixing this before approval

Comment (nerdlet) [reply src/components/Mapbox.tsx]: @karimkawambwa Done, otherwise everything looks good

**Pull request merged**